Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/README.md

```diff
@@ -0,0 +1,274 @@
+SaaS Security is an integrated CASB (Cloud Access Security Broker) solution that helps Security teams like yours meet 
+the challenges of:
+* protecting the growing availability of sanctioned and unsanctioned SaaS applications
+* maintaining compliance consistently in the cloud
+* stopping threats to sensitive information, users, and resources
+
+
+## Configure SaaS Security on Cortex XSIAM
+
+1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automations & Feed Integrations**.
+2. Search for Saas Security Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Your server URL | The instance configuration URL based on the server location. | True |
+    | Client ID | The SaaS Security Client ID. | True |
```

Comment:
```suggestion
    | Client ID | The Saas Security Client ID. | True |
```

Reply:
i think they only meant to the display on the integration in the yml, no need to change it i guess